[DO NOT MERGE] Add docs for how project deletion works for ingested resources (#2422)
chef/automate: components/automate-chef-io/content/docs/iam-v2-guide.md

Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.

#### Deleting Projects

> To delete a project, navigate to the Projects list under the **Settings** tab. Find the project you wish to delete in the list, click the menu on the

Reviewer: Do we normally capitalize resources?

Reviewer: mmm, reading the paragraph above, looks like we refer to the "project list page" instead of "Projects list"

Author: Looks like no on resource capitalization.

end of the row, and select `Delete Project`.

When a project is deleted, any resources in that project are removed from that project. For ingested resources, it could take several hours

> for every resource to be removed from the project. If the resource was only in the deleted project then that resource will become unassigned.

Reviewer 2: Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>
```suggestion
for every resource to be removed from the project. If the resource was only in the deleted project, then that resource will become unassigned.
```

> for the project to be removed from every resource. If the project delete was the only project assigned to a resource, they become unassigned.

Reviewer 3: ```suggestion
for every resource to be removed from the project. If the resource was only in the deleted project then it will become unassigned.
```

> for the project to be removed from every resource. If the project delete was the only project assigned to a resource, they become unassigned.

Reviewer: ```suggestion
for every resource to be removed from the project. If the resource was only in the deleted project then that resource will become unassigned.
```
